src/main/groovy/com/athaydes/gradle/ceylon/CeylonPlugin.groovy

```diff
@@ -99,75 +100,82 @@ class CeylonPlugin implements Plugin<Project> {
                 description: 'Import transitive Maven dependencies and copies the output from dependent Ceylon ' +
                         'projects into the local Ceylon repository .\n' +
                         'To enable importing Maven dependencies, the Ceylon config property "importJars" must ' +
                         'be set to true.',
-                'importJars' )
+                'importJars')
 
         Task compileTask = project.task(
                 type: CompileCeylonTask,
                 group: 'Build tasks',
-                dependsOn: [ 'generateOverridesFile', 'importJars', 'createMavenRepo' ],
+                dependsOn: ['generateOverridesFile', 'importJars', 'createMavenRepo'],
                 description: 'Compiles Ceylon and Java source code and directly' +
                         ' produces module and source archives in a module repository.',
-                'compileCeylon' )
+                'compileCeylon')
 
         project.task(
                 type: RunCeylonTask,
                 group: 'Verification tasks',
                 dependsOn: 'compileCeylon',
                 description: 'Runs a Ceylon module.',
-                'runCeylon' )
+                'runCeylon')
 
         project.task(
                 type: CompileCeylonTestTask,
                 group: 'Build tasks',
-                dependsOn: [ 'compileCeylon' ],
+                dependsOn: ['compileCeylon'],
                 description: 'Compiles Ceylon and Java test code and directly' +
                         ' produces module and source archives in a module repository.',
-                'compileCeylonTest' )
+                'compileCeylonTest')
 
         Task testTask = project.task(
                 type: TestCeylonTask,
                 group: 'Verification tasks',
-                dependsOn: [ 'compileCeylonTest' ],
+                dependsOn: ['compileCeylonTest'],
                 description: 'Runs all tests in a Ceylon module.',
-                'testCeylon' )
+                'testCeylon')
 
         project.task(
                 type: CreateJavaRuntimeTask,
                 group: 'Build tasks',
-                dependsOn: [ 'compileCeylon' ],
+                dependsOn: ['compileCeylon'],
                 description: 'Creates a Java runtime containing the full classpath' +
                         ' as well as bash and bat scripts that can be used to invoke Java to run the Ceylon module' +
                         ' in a JVM without any Ceylon tool.',
-                'createJavaRuntime' )
+                'createJavaRuntime')
+
+        project.task(
+                type: FatJarTask,
+                group: 'Build tasks',
+                dependsOn: ['compileCeylon'],
+                description: 'Creates fat jar jarOut option can be added to configuration to make task deploy jar on specific output path.',
+                'fatJar')
```

Comment:
if that does not work, try changing the version of the project in the main build file, then change the version in the test parent build file as well, and start again. That surely must work.